nixos/modules/system/boot/plymouth.nix

```diff
@@ -200,13 +220,13 @@ in
     systemd.services.emergency = preStartQuitFixup;
 
     boot.initrd.systemd = {
-      extraBin.plymouth = "${plymouth}/bin/plymouth"; # for the recovery shell
+      extraBin.plymouth = "${cfg.package}/bin/plymouth"; # for the recovery shell
       storePaths = [
-        "${lib.getBin config.boot.initrd.systemd.package}/bin/systemd-tty-ask-password-agent"
-        "${plymouth}/bin/plymouthd"
-        "${plymouth}/sbin/plymouthd"
+        "${getBin config.boot.initrd.systemd.package}/bin/systemd-tty-ask-password-agent"
+        "${cfg.package}/bin/plymouthd"
```

Comment:
Why are both the bin/plymouthd and sbin/plymouthd paths included? Do we need both?